cluster/pre-existing/util.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
```

Comment:
We would have it included through https://github.com/kubernetes/kubernetes/blob/master/test/kubemark/start-kubemark.sh#L24 in start-kubemark script.